package.json

```diff
@@ -73,7 +73,7 @@
     "svgo": "^3.3.2",
     "table": "^6.8.1",
     "typescript": "5.3.3",
-    "umberto": "^9.0.0",
+    "umberto": "github:cksource/umberto#performance-research-2-with-css-select",
```

Comment:
This needs to be reverted together with changes in `pnpm-lock.yaml` before merging.